jina/drivers/dbms.py

```diff
@@ -0,0 +1,27 @@
+from jina.drivers.index import BaseIndexDriver
+from .. import Document
+
+if False:
+    from ..types.sets import DocumentSet
+
+
+# TODO move?
+def doc_without_embedding(d):
+    new_doc = Document()
```

Comment:
Document offers the option to pass a `document` and do `copy` in the constructor

Reply:
Yes, good point. I refactored this method use the constructor with copy=True directly.